README.md

```diff
@@ -14,29 +14,23 @@ Our chat room is `#skonfig:matrix.org` (you need [Matrix client](https://matrix.
 
 ## Getting Started
 
-There will be no versioned releases fow now - everything is expected to be used from the repositories.
-
-We are currently targeting (`cdist`) power users, so expect some rough edges.
-
 ### Step 1: Install `skonfig`
 
 ```sh
-git clone https://github.com/skonfig/skonfig
-ln -s "$PWD/skonfig/bin/skonfig" "$HOME/.local/bin/skonfig"
+pip3 install skonfig@git+https://github.com/skonfig/skonfig
 ```
 
-### Step 2: Create configuration directory and initial manifest
+### Step 2: Clone base and extra sets repositories
```

Comment:
Just "Clone sets repositories"?

Reply:
OK. I added a sentence describing why we recommend to install the base and extra sets.